src/style.scss

```diff
@@ -72,9 +89,9 @@
 
 .helpText {
   margin-top: 4px;
```

Comment:
should I apply the tokens also for this hardcoded value?

Reply:
Not right now, we have a body of work coming to replace all remaining styles that have hard values.